react/features/base/flags/constants.js

```diff
@@ -155,6 +155,12 @@ export const RECORDING_ENABLED = 'recording.enabled';
  */
 export const RESOLUTION = 'resolution';
 
+/**
+ * Flag indicating if the security options button should be enabled.
+ * Default: enabled (true)
+ */
+export const ENABLED_SECURITY_OPTIONS = 'security-options.enabled';
```

Comment:
alpha sort please. also, I don't see this used anywhere?

Reply:
Sorted.